cli.js

```diff
@@ -73,8 +73,25 @@ function filterProcesses(input, processes, flags) {
 		});
 }
 
+function handleFkillError(processes) {
+	const suffix = processes.length > 1 ? 'es' : '';
+	if (process.stdout.isTTY === false) {
+		console.log(`Error killing process${suffix}. Try \`fkill --force ${processes.join(' ')}\``);
+	} else {
+		return inquirer.prompt([{
+			type: 'confirm',
+			name: 'forceKill',
+			message: `Error killing process${suffix}. Would you like to use the force?`
```

Comment:
I don't like adding things just in case it's needed later. That just leads to lot of unused cruft in the codebase. We add things when we actually need it. I actually noticed this because it also returns the wrong value, `processes`.